multiapps-controller-persistence/src/main/resources/org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog.xml

```diff
@@ -16,7 +16,10 @@
         file="org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog-1.116.0-persistence.xml" />
     <include file="org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog-1.110.0-core.xml" />
     <include file="org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog-1.116.0-core.xml" />
+
     <include file="org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog-1.120.0-quartz.xml" />
     <include file="org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog-1.120.0-core.xml" />
 
+    <include file="org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog-1.121.0-core.xml" />
```

Comment:
Is there any leftovers from the old spring library, which might have created any data in our database? If yes, then there should be a liquibase changelog for the removal.

Reply:
Yes, there's a table oauth_access_token, but I plan to remove it after a couple of tacts just for backwards compatibility in case we revert the change or the version of ds